.github/workflows/ci.yml

```diff
@@ -15,10 +15,8 @@ jobs:
         python-version: ["3.8"]
         toxenv:
           [
-            celery44-django32,
-            celery52-django32,
-            celery44-django40,
-            celery52-django40,
```

Comment:
Keep `celery44-django32` and `celery52-django32` as we currently have these versions in our edX services.